pkg/auth/auth.go

```diff
@@ -20,6 +20,10 @@ import (
 	"context"
 )
 
+const (
```

Comment:
keeping this as an unexported constant, as importing the role pkg creates a cyclic dependency.